### ISSUE TITLE
Make window resizeable (Trac #140)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#144]
Migrated from http://microfluidics.utoronto.ca/microdrop/ticket/140
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "Make window resizeable", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2013-04-19T00:29:26", 
    "milestone": "Microdrop 1.0", 
    "owner": "", 
    "type": "enhancement"
}
```

**Comment by cfobel** [comment]
Trac update at 20130513T21:02:01: 
* ryan commented:

> Fixed in commit 28ae3962a08c24c475e08f5b40b08c5920efdda5


* ryan changed resolution from "" to "fixed"
* ryan changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "" to "microdrop"
